packages/ms-authenticator/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@sphereon/ms-authenticator",
+  "version": "0.0.1",
```

Comment:
Please make the version the same as the parent version. This is a multi-module workspace with lerna. Versions are not managed individually

Reply:
changed